cfme/infrastructure/openstack_node.py

```diff
@@ -2,7 +2,13 @@
 """A model of an Openstack Infrastructure Node in CFME."""
 
 from cfme.infrastructure.host import Host
-from cfme.utils.appliance.implementations.ui import navigate_to
+import attr
```

Comment:
**Required**

Import statements need to be organized and alphabetized:
http://cfme-tests.readthedocs.io/guides/dev_guide.html#code-style